tox.ini

```diff
@@ -18,7 +18,7 @@ deps =
     sphinx
     sphinx-rtd-theme
     -rdev-requirements.txt
-commands=make html
+commands=/usr/bin/make html
 
```

Comment:
I'm checking whether we want to keep this since we don't use it. Either way putting a full path in the remote seems questionable to me.